### DISCUSSION TITLE
Proxmox Helper Script - Homeassistant SSH login ?

### DISCUSSION THREAD

**Discussion by Nexow91** [tteck/Proxmox#3141] [Help Needed] [answered]
Hello everyone, 

I am using the Proxmox Helper Script for Homeassistant (VM) so far so good. Unfortunately I can't get into the VM via SSH (Putty) because the script didn't set a root password or user password during the automatic installation. 

How can I now access the VM via SSH? Via VNC Proxmox console I am in the Homeassistant CLI from there but I can't change a password with “passwd”. 

https://helper-scripts.com/scripts?id=Home+Assistant+OS

Is there a default login via SSH ?

**Comment by tteck** [comment, marked as answer]
https://www.home-assistant.io/common-tasks/os/#installing-and-using-the-ssh-add-on
